ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -606,6 +606,12 @@ def isclose(self, y, rtol=1e-05, atol=1e-08, equal_nan=False, name=None):
     def floor_divide(self, y, name=None):
         return paddle_frontend.floor_divide(self, y)
 
+    @with_supported_dtypes(
+        {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle"
+    )
+    def floor_mod(self, y, name=None):
+        return paddle_frontend.Tensor(ivy.remainder(self._ivy_array, _to_ivy_array(y)))
```

Comment:
Hey @AbdullahSabry 
no need to use casting methods as array is already `ivy`. so, you can use `paddle_frontend.remainder(self, y)` directly.
thanks!

Reply:
@abdulasiraj 
Yes you're right